stencil/src/components/pulumi-tertiary-nav/pulumi-tertiary-nav.tsx

```diff
@@ -0,0 +1,64 @@
+import { Component, Host, h, Prop } from "@stencil/core";
+
+export interface TertiaryNavItem {
+  // The anchor for the content that the nav item controls viewability of
+  anchor: string;
+  // The text that should show for the nav item
+  label: string;
```

Comment:
Stumbled a bit on the wording, and channeling my inner chrsmith on the punctuation 😄:

```suggestion
    // The anchor of the content that the nav item shows.
    anchor: string;
    // The text that should show for the nav item.
    label: string;
```